src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js

```diff
@@ -2642,38 +2650,45 @@ import { getLowerAgeYears } from '@ncigdc/utils/ageDisplay';
     });
   };
 
-  InCHlib.prototype._highlight_cluster = function (path_id) {
+  InCHlib.prototype._highlight_cluster = function (path_id, id) {
```

Comment:
What's the difference between those two ids?

Reply:
the `path_id` is used to construct the `id` and doesn't contain all of the same information - in this case it doesn't contain `_top_node` for the top dendrogram node

Reply:
meaning, the `path_id` is sort of the original, and then the id would be the one actually used in the DOM.
is that correct?

Reply:
something like that. i figured it was safer to customize the constructed, final ID than its constituent parts.